en/intro/conventions.rst

```diff
@@ -42,6 +42,16 @@ the convention is that your URLs are lowercase and dashed using the
 ``DashedRoute`` class, therefore ``/red-apples/go-pick`` is the correct form
 to access the ``RedApplesController::goPick()`` action.
 
+When you create links using ``this->Html->link()``, you can use the following
+conventions for the url array::
+
+    $this->Html->link('link-title', [
+        'prefix' => 'my-prefix // unmodified
+        'plugin' => 'MyPlugin, // camelCased
+        'controller' => 'ControllerName', // camelCased
```

Comment:
I'd rather write it ``CamelCased`` (with a capital C) to better reflect the "format".